.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,26 @@
+---
+name: User story
+about: Provide a user story to help the app's development.
+
+---
+
+## Description
+As a [USER],
+I need [TO DO THIS],
+so that I can [ACCOMPLISH THAT].
+
+## Mocks
```

Comment:
Will a typical, non tech-savyy user know what to put here?

Reply:
This has the description below `[INSERT RELEVANT PNG FILE]`. What do you suggest here @bartekpacia so @Sanika1999 knows what is the change requested.

Reply:
I'd change "mocks" to simply "screenshots of the issue", it's more clear to a typical person in my opinion.